backend/tests/ops_site/cans/test_models.py

```diff
@@ -89,3 +90,22 @@ def test_ContractLineItemFiscalYear_name():
     )
 
     assert contract_line_item_for_2022.name == contract_line_item_name
+
+
+@pytest.mark.django_db
+def test_FundingPartner_str():
+    fundingPartner = FundingPartner.objects.create(
+        name="FundingPartnerName", nickname="NotTheRealName"
+    )
+    assert "FundingPartnerName" == str(fundingPartner)
```

Comment:
Just a dumb nit, take it or leave it: I like to make a variable that is set to `"FundingPartnerName"`.  I then use that variable in the creation of the `FundingPartner` and in the `assert`.  It makes the test more resilient and calls out how the set-up is linked with the expected output.

This could apply to the test below.

```suggestion
def test_FundingPartner_str():
    name = "FundingPartnerName"
    fundingPartner = FundingPartner.objects.create(
        name=name, nickname="NotTheRealName"
    )
    assert name == str(fundingPartner)
```